metis-normalization/src/main/java/eu/europeana/normalization/dates/DateNormalizationResult.java

```diff
@@ -95,6 +96,7 @@ private boolean isEitherDateNonPrecise(EdtfDatePart startEdtfDatePart, EdtfDateP
         || startEdtfDatePart.getYearPrecision() != null || endEdtfDatePart.getYearPrecision() != null;
   }
 
+
   // TODO: 22/07/2022 Is this calculation correct??
```

Comment:
So this computation should be with "equals" to null for both the days and months(instead of not equals as it is now).
The order is not significant.